src/screens/ChangeWalletSheet.js

```diff
@@ -186,7 +186,7 @@ export default function ChangeWalletSheet() {
       );
       if (visibleAddresses.length === 0) {
         delete newWallets[walletId];
-        await dispatch(walletsUpdate({ ...newWallets, [walletId]: undefined }));
+        await dispatch(walletsUpdate({ ...newWallets }));
```

Comment:
```suggestion
        await dispatch(walletsUpdate(newWallets));
```

There's no need to reiterate